src/platform_impl/web/window.rs

```diff
@@ -246,6 +246,10 @@ impl Window {
         false
     }
 
+    pub fn is_resizable(&self) -> bool {
```

Comment:
some are `#[inline]` while others not. Is there a logic ?

Reply:
Just a typo on my end. I just follow the convention of the platforms I didn't implement